build.proj

```diff
@@ -73,12 +83,20 @@
           WorkingDirectory="$(DotnetCliBuildDirectory)"/>
   </Target>
 
+  <Target Name="WriteDynamicPropsToStaticPropsFiles"
```

Comment:
From here down, I'd move these changes to an `InitRepo.targets` and just import that project here.